static/networks/mainnet.json

```diff
@@ -39,16 +39,12 @@
       "Name": "Status page and incidents",
       "Link": "https://filecoin.statuspage.io"
     },
-    {
-      "Name":"Faucet",
-      "Link":"https://spacerace.faucet.glif.io/"
-    },
     {
       "Name":"Stats Dashboard",
-      "Link":"https://stats.testnet.filecoin.io/"
+      "Link":"https://stats.filecoin.io/"
     },
     {
-      "Name":"Slack Channel: #fil-testnet",
+      "Name":"Slack Channel: #fil-testnet-announce",
```

Comment:
What is this suppose to be now? I assume it shouldn't be testnet.

Reply:
we'll fix that later when these are renamed.